lib/config.js

```diff
@@ -14,6 +14,7 @@ function defaults() {
   CONFIG.ACTION_DELETE_OBJECT = false;
   CONFIG.MAX_ATTEMPTS = 10;
   CONFIG.MAX_ATTEMPT_DELAY_MSEC = 30_000;
+  CONFIG.SIGNED_URL_DURATION = 3600
```

Comment:
Once this is merged, please update https://github.com/uvasoftware/scanii-lambda/wiki/Configuration